e107_languages/English/admin/lan_plugin.php

```diff
@@ -306,7 +306,7 @@
 define("LAN_INSTALL_FAIL", "Installation failed!");
 define("LAN_UNINSTALL_FAIL", "Unable to uninstall!");
 define("LAN_PLUGIN_IS_USED", "[x] plugin is used by:");
-
+define("LAN_PATH", "Path");
 
```

Comment:
It should be in the non-universal format: EPL_ADLAN_XX 
The ONLY files which should contain the non-numeric format should be lan_admin.php and e107_languages/English/English.php . Please keep this in mind when adding new LANs. Thank you!